src/snowflake/cli/plugins/notebook/manager.py

```diff
@@ -48,19 +48,19 @@ def parse_stage_as_path(notebook_file: NotebookName) -> Path:
 
     def create(
         self,
-        notebook_name: NotebookName,
+        notebook_name: FQN,
         notebook_file: NotebookStagePath,
     ) -> str:
-        notebook_fqn = FQN.from_string(notebook_name).using_connection(self._conn)
+        notebook_fqn = notebook_name.using_connection(self._conn)
         stage_path = self.parse_stage_as_path(notebook_file)
 
         queries = dedent(
             f"""
-            CREATE OR REPLACE NOTEBOOK {notebook_fqn.identifier}
+            CREATE OR REPLACE NOTEBOOK {notebook_fqn.sql_identifier}
             FROM '{stage_path.parent}'
             QUERY_WAREHOUSE = '{get_cli_context().connection.warehouse}'
             MAIN_FILE = '{stage_path.name}';
-
+            // Cannot use IDENTIFIER(...)
```

Comment:
Is this comment needed in query?

Reply:
Well, that was the best option. Alternative is to split the query text into two and leave comment in python. WDYT?